meta/regular_expression.rb

```diff
@@ -0,0 +1,14 @@
+Mutant::Meta::Example.add :'regexp_expression' do
+  source '/(foo|bar)/'
```

Comment:
Just as a note, I'll probably do a simpler mutation like `^` -> `\A`. The representation for alternations is actually pretty complex compared to simple nodes like the beginning of string anchor